locales/en-US/strings.properties

```diff
@@ -146,6 +146,13 @@ topsites_form_add_button=Add
 topsites_form_save_button=Save
 topsites_form_cancel_button=Cancel
 topsites_form_url_validation=Valid URL required
+topsites_form_label_title=Title
+topsites_form_label_url=URL
+topsites_form_label_screenshot_url=Custom Image
+topsites_form_screenshot_request_error=Image failed to load. Try a different URL.
+topsites_form_activate_custom_screenshot=Use a custom image
+topsites_form_custom_screenshot_placeholder=Paste a URL here
```

Comment:
Why does one placeholder say "Type or paste" and the other is just "Paste"? Can't you type into the topsites URL?

```
topsites_form_url_placeholder=Type or paste a URL
topsites_form_custom_screenshot_placeholder=Paste a URL here
```

Reply:
I'll bring this up for discussion.